data/test/scenarios/test_return.cfg

```diff
@@ -15,3 +15,17 @@
         {RETURN ([not][true][/true][/not])}
     [/event]
 )}
+
+# Same as the above two, but using the Lua unit_test module
+{GENERIC_UNIT_TEST "test_lua_return" (
+    [event]
+        name = start
+        code=<<unit_test.pass_if(true, 'return true')>>
+    [/event]
+)}
+{GENERIC_UNIT_TEST "test_lua_return_fail" (
+    [event]
+        name = start
+        code=<<unit_test.pass_if(false, 'return false')>>
```

Comment:
I'd prefer `unit_test.succeed()` for the success cases, and replacing the `unit_test.pass_if(err:find('test')` with `unit_test.assert_something(err:find('test'))`.

For WML it made sense to have macros for conditionals, but that's because the WML syntax for conditional statements is bad.

Reply:
So basically, remove `pass_if` and `fail_if`? The `pass_if` function was intended to be equivalent to the `{RETURN}` macro.